lib/ext/heroku/command/addons.rb

```diff
@@ -439,9 +464,12 @@ def open
         end
       when 1 then
         addon_to_open = matches.first
-        launchy("Opening #{addon_to_open} for #{app}", app_addon_url(addon_to_open))
+        launchy("Opening #{addon_to_open["name"]} for #{app}", addon_to_open["web_url"])
```

Comment:
The commands you used only work because you only have a single instance of each of those add-ons. Once `many_per_app` comes into play, you'd need to use the resource name. I suppose showing the service still makes sense though, since you're opening the service, not the addon. :+1: